File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/witnesses/F10.P2/go/c3b/w.go
package main

import "fmt"

const i = 1e400

func probe() float64 {
	var d float64 = 0.5
// BEGIN PROBE F10.P2
sum := float64(i) + d
// END PROBE F10.P2
	return sum
}

func main() { fmt.Println(probe()) }

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/kotlin/chunk_B/f10p2_V2_same_int/w.kt
// F10.P2 kotlin WITNESS V2 (both operands the same fixed-width integer type).
fun probe(i: Int, d: Int): Int {
    // BEGIN PROBE F10.P2
    val sum = i + d
    // END PROBE F10.P2
    return sum
}
fun main() { println("sum=" + probe(3, 1) + " overflow=" + probe(Int.MAX_VALUE, 1)) }
